tests/DeSerializationTest.php

```diff
@@ -53,7 +56,7 @@ public function export($value)
         ];
         foreach ($rc->getProperties() as $prop) {
             $prop->setAccessible(true);
-            $v = $prop->isInitialized($value) ? $prop->getValue($value) : null;
+            $v = $prop->isInitialized($value) ? $prop->getValue($value) : self::UNINITIALIZED;
```

Comment:
I copied the behaviour from the `Php81DeSerializationTest` class. I think it's better to be more explicit about the returned value so the tests are self-documenting